pkgs/applications/editors/vscode/extensions/chenglou92.rescript-vscode/rescript-editor-analysis.nix

```diff
@@ -1,38 +1,28 @@
 {
   lib,
-  stdenv,
   fetchFromGitHub,
-  ocaml,
   ocamlPackages,
-  dune_3,
   version,
```

Comment:
I would suggest to remove the version parameter, or replace it with version and hash. Otherwise it is impossible to use correctly. I can take care of it after this PR.

Reply:
This is removed? Or are you talking about something else?

Reply:
Oh I see